docs/docs/04-configuration/03-auth-providers.md

```diff
@@ -1,2 +1,30 @@
 # Auth Providers
-Include details about how to configure each supported auth provider
\ No newline at end of file
+
+Authentication providers allow your Otto installation to authenticate users with the identity provider of your choice. All authentication providers are configured using environment variables.
+
+## Common Environment Variables
+
+The following environment variables are required for all authentication providers. Setting the Client ID and Client Secret will mean that the authentication provider is enabled. The remaining configuration will be validated on startup.
+
+- `OTTO_AUTH_CLIENT_ID`: The client ID of the authentication provider.
+- `OTTO_AUTH_CLIENT_SECRET`: The client secret of the authentication provider.
+- `OTTO_AUTH_COOKIE_SECRET`: The secret used to encrypt the authentication cookie. Must be of size 16, 24, or 32 bytes.
+- `OTTO_AUTH_ADMIN_EMAILS`: A comma-separated list of the email addresses of the admin users.
+
+The following environment variables are optional for all authentication providers:
+- `OTTO_AUTH_EMAIL_DOMAINS`: The email domains allowed for authentication. Defaults to `*`.
+- `OTTO_AUTH_CONFIG_TYPE`: The type of the authentication provider. For example, `google` or `github`. Defaults to `google`.
+
+## Google
+
+Google is the default authentication provider. There are currently no additional environment variables required for Google authentication.
+
+## GitHub
+
+GitHub authentication has the following optional configuration:
+
+- `OTTO_AUTH_GITHUB_ORG`: The name of the organization allowed for authentication. Defaults to `*`.
+- `OTTO_AUTH_GITHUB_TEAM`: The name of the team allowed for authentication. Defaults to `*`.
```

Comment:
Are these also allowed to be comma-separated lists? The wording below for `OTTO_AUTH_GITHUB_ALLOW_USERS` makes it seem like it

Reply:
No, this can't be a comma-separated list. I changed the wording to make it explicit.